### PR TITLE
fixed order of bars for barh

### PR DESCRIPTION
**Pull request by ericz82**
Turns out pyplot.barh plots bars starting from the bottom to the top

Fixes issue #58

### CODE REVIEW

File: datascience/tables.py [hunk 1]
```diff
@@ -888,10 +888,10 @@ def draw(axis, label, color):
                 ypos = index + margin + (1-2*margin)*labels.index(label)/len(labels)
             else:
                 ypos = index
-            axis.barh(ypos, self[label], width,  color=color, **options)
+            axis.barh(ypos, self[label][::-1], width,  color=color, **options)
         def annotate(axis, ticks):
             axis.set_yticks(index+0.5) # Center labels on bars
-            axis.set_yticklabels(ticks, stretch='ultra-condensed')
+            axis.set_yticklabels(ticks[::-1], stretch='ultra-condensed')
```
Comment (SamLau95) [inline datascience/tables.py]: This is a bit unintuitive for someone reading the code for the first time. Can you add a comment here saying something like the default behavior is to display in reverse order which is why we reverse again here?

Comment (ericz82) [inline datascience/tables.py]: Yeah, you want me to make another pull request?

Comment (SamLau95) [inline datascience/tables.py]: No worries, just commit and push to master for small changes like that.
